DP-18/Add basic troubleshooting section to the docs (#5901)
OpenNMS/opennms: docs/modules/troubleshooting/pages/other-issues.adoc

   </event>
           ^
/opt/opennms/etc/events/VMWare.events.xml:276: parser error : Opening and ending tag mismatch: logmsg line 265 and events
</events>
         ^
/opt/opennms/etc/events/VMWare.events.xml:277: parser error : Premature end of data in tag event line 223
^
----

In this example, an unmatched tag in `/opt/opennms/etc/events/VMWare.events.xml` is the source of the issue:

[source, xml]
----
265      <logmsg dest="logndisplay">&lt;p>
266            vpxdTrap trap received
267            vpxdTrapType=%parm[#1]%
268            vpxdHostName=%parm[#2]%
269            vpxdVMName=%parm[#3]%
270            vpxdNewStatus=%parm[#4]%
271            vpxdOldStatus=%parm[#5]%
272            vpxdObjValue=%parm[#6]%&lt;/p>
273      <logmsg>
----

This error can be fixed by updating line 273 to `</logmsg>`.
If you run `xmllint` again, the file should validate properly:

[source, shell]
----
# xmllint --schema /var/opennms/xsds/eventconf.xsd /opt/opennms/etc/events/VMWare.events.xml --noout
/opt/opennms/etc/events/VMWare.events.xml validates
----

{page-component-title} should now start properly.

> You can run `xmllint` on all files in `$\{OPENNMS_HOME}/etc/events` with the following command:

Reviewer: ```suggestion
You can run `xmllint` on all files in `$\{OPENNMS_HOME}/etc/events` using this command:
```
